packages/block-library/src/blocks/header/edit.js

```diff
@@ -98,6 +109,10 @@ class Edit extends Component {
 		}
 	}
 
+	componentDidUpdate() {
+    addSocialMenuClass.call(this);
```

Comment:
`addSocialMenuClass` doesn't have any usages of `this` rendering the usage of the `call` method useless.
`addSocialMenuClass()` should suffice.

Reply:
This could be optimised because `componentDidUpdate` is triggered often. However, firstly the Edit component should be rewritten using hooks.